e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -724,6 +724,49 @@ def test_set_workspace_namespace_bindings(
     output = detproc.check_output(sess, set_binding_cmd + ["--namespace", namespace])
     assert bound_to_namespace in output
 
+    # MultiRM: Valid cluster name, no namespace name, auto-create namespace & resource_quota.
+    if is_multirm_cluster:
+        w_name = uuid.uuid4().hex[:8]
+        output = detproc.check_output(
+            sess,
+            [
+                "det",
+                "w",
+                "create",
+                w_name,
+                "--cluster-name",
+                conf.DEFAULT_RM_CLUSTER_NAME,
+                "--auto-create-namespace",
+                "--set_resource_quota-quota",
+                "1",
+            ],
+        )
+        assert bound_to_namespace in output
+    # SingleRM: No cluster name, no namespace name, auto-create namespace & resource quota.
+    else:
+        w_name = uuid.uuid4().hex[:8]
+        output = detproc.check_output(
+            sess,
+            [
+                "det",
+                "w",
+                "create",
+                w_name,
+                "--auto-create-namespace",
+                "--set_resource_quota-quota",
```

Comment:
```suggestion
                "--resource-quota",
```
same for other commands like this!